dev-python/argon2-cffi/argon2-cffi-21.2.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} pypy3 )
+DISTUTILS_USE_SETUPTOOLS=pyproject.toml
+inherit distutils-r1
+
+DESCRIPTION="CFFI bindings to the Argon2 password hashing library"
+HOMEPAGE="https://github.com/hynek/argon2-cffi"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+DEPEND=">=dev-python/argon2-cffi-bindings-21.2.0"
```

Comment:
Does it actually need an explicit DEPEND though? I thought this was supposed to be a pure Python package now.

Reply:
That's correct. I removed the bindings from DEPEND, leaving them only in RDEPEND. I also added the missing `PYTHON_USEDEP`.